jdaviz/core/marks.py

```diff
@@ -59,11 +59,79 @@ def _update_counts(self, *args):
         self.right.text = [f'{oob_right} \u25b6' if oob_right > 0 else '']
 
 
-class BaseSpectrumVerticalLine(Lines, HubListener):
+class PluginMark():
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
Wait, this doesn't inherit anything...does the base python `Class` init handle setting `self.[whatever]` for kwargs if you call `super().__init__` like this?

Reply:
turns out its required in order for the `comm` to be set correctly when other classes inherit from this and `Lines`